src/test/ui/feature-gate/feature-gate-const_refs_to_cell.rs

```diff
@@ -0,0 +1,12 @@
+// check-pass
+
+#![feature(const_refs_to_cell)]
+
+const FOO: () = {
+    let x = std::cell::Cell::new(42);
+    let y = &x;
+};
```

Comment:
Which is the test ensuring that this is *not* accepted without the feature gate?

Reply:
That would be `src/test/ui/consts/const-multi-ref.rs` (you can see it in this PR as its message slightly changed)